staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/well_known_labels.go

```diff
@@ -26,6 +26,12 @@ const (
 	LabelOS   = "beta.kubernetes.io/os"
 	LabelArch = "beta.kubernetes.io/arch"
 
+	// When kubelet is started with the "external" cloud provider, then
+	// it sets this label on the node to denote an ip address set from the
+	// cmd line flag. This ip is verified with the cloudprovider as valid by
+	// the cloud-controller-manager
+	LabelProvidedIPAddr = "beta.kubernetes.io/provided-node-ip"
```

Comment:
I'm still not clear why this is a label rather than annotation?  Maybe I missed the explanation.  Any why is it beta?  Will it become a field?